### DISCUSSION TITLE
PrimeNG multiselect filter issue

### DISCUSSION THREAD

**Discussion by lakshmi2716** [primefaces/community#45] [PrimeNG] [answered]
https://primeng.org/multiselect#filter . on select all with filter, previously selected items are cleared. 
 
 Issue mentioned in https://github.com/primefaces/primeng/issues/7047 still persists.

**Comment by yigitfindikli** [comment, marked as answer]
Hi, @lakshmi2716!

Thanks a lot for your investigation. Sounds right but all selectable prime components have this behaviour (I'm thinking the backward compatibility.). Maybe we can add an optional feature something like headerSelectClearBefore. 

I created a Feature Request for you. You can track at https://github.com/primefaces/primeng/issues/13678.

**Reply by MORUNANI** [reply]
Hi,@yigitfindikli this issue still not resiolve ?

**Reply by yigitfindikli** [reply]
Hi @MORUNANI, 

Feature requests are implemented by priority. This feature request is not planned yet.

**Reply by MORUNANI** [reply]
thanks for your update
